micropip/_utils.py

```diff
@@ -62,8 +62,13 @@ def get_files_in_distribution(dist: Distribution) -> set[Path]:
 
 
 @functools.cache
-def sys_tags() -> list[Tag]:
-    return list(sys_tags_orig())
+def sys_tags() -> tuple[Tag, ...]:
+    new_tags = []
+    for tag in sys_tags_orig():
+        if "emscripten" in tag.platform:
+            new_tags.append(Tag(tag.interpreter, tag.abi, "pyodide_2024_0_wasm32"))
```

Comment:
I think you can use

```
import sysconfig
abi_version = sysconfig.get_config_var("PYODIDE_ABI_VERSION")
tag = f"pyodide_{abi_version}_wasm32"
```

instead, as abi version is expected to be changed when Pyodide ABI changes.